ibm/service/power/resource_ibm_pi_network_port_attach.go

```diff
@@ -72,7 +72,7 @@ func ResourceIBMPINetworkPortAttach() *schema.Resource {
 			"port_id": {
 				Type:       schema.TypeString,
 				Computed:   true,
-				Deprecated: "port_id attribute is deprecated, use network_port_id instead.",
+				Deprecated: "This field is deprecated, use network_port_id instead",
```

Comment:
same as above -- Since it is deprecated , we might as well remove it.

Reply:
Removed deprecated fields.